accounts/templates/account/login.html

```diff
@@ -0,0 +1,32 @@
+{% extends "exp/base.html" %}
+
+{% load i18n %}
+{% load account socialaccount %}
+{% load bootstrap3 %}
+
+{% block head_title %}{% trans "Sign In" %}{% endblock %}
```

Comment:
I think that makes sense. There's a view at `/exp/` that just redirects to `/exp/studies/`. That could be changed to conditionally redirect based on the user having permissions and then toss a template in that says you've not got any permissions and you'll be notified when someone gives you some. Make sense?